requests/pulls and committs.py

```diff
@@ -0,0 +1,109 @@
+import requests
```

Comment:
VERIFICATION RESULT/nYour pull request REQUESTS-1022 Added pulls and commits.py/n

Your commit: REQUESTS-1022 Added pulls and committs.py

Your commit: REQUESTS-1022 Refactored pulls and committs.py

Your commit: REQUESTS-1022 Refactored pulls and committs.py

Your commit: REQUESTS-1022 Refactored pulls and committs.py

Your commit: REQUESTS-1022 Refactored pulls and commits.py

Your commit: REQUESTS-1022 Refactored pulls and commits.py

Your commit: Update pulls and committs.py
You need to write "-" between prefix of your message and group number
Your commit: REQUESTS-1022 pulls and commits
The action of your message must contain action in ['Added', 'Deleted', 'Refactored', 'Fixed', 'Moved']
Your commit: REQUESTS-1022 Refactored pulls and commits

Your commit: REQUESTS-1022 Refactored pulls and commits

Check last comment

Your commit: REQUESS-1024 Aded pelfl
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE', 'REQUESTS']/nThe group number of your message must contain group number in ['1011', '1012', '1021', '1022']/nThe action of your message must contain action in ['Added', 'Deleted', 'Refactored', 'Fixed', 'Moved']